app/src/main/java/com/github/studenttimetracker/MainActivity.java

```diff
@@ -45,6 +49,22 @@ protected void onCreate(Bundle savedInstanceState) {
         NavigationView leftNav = findViewById(R.id.left_navigation);
         leftNav.setNavigationItemSelectedListener(leftNavListener);
 
+        // Filling database with data
+        if(!this.getDatabasePath(DatabaseHelper.DATABASE_NAME).exists()) {
+            try {
+                SQLiteDatabase database = new DatabaseHelper(this).getWritableDatabase();
+                InputStream inputStream = getResources().getAssets().open("databaseSetup.sql");
+                BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
+                while (bufferedReader.ready()) {
+                    // In case of error: check if databaseSetup.sql contains no-SQL lines. Like empty lines, comments (yes even sql comments!)
+                    database.execSQL(bufferedReader.readLine());
+                }
+                bufferedReader.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+
```

Comment:
Robienie tego tutaj nie jest imo super eleganckie, ale jak robiłem w DatabaseHelper#OnCreate() to nie mogło mi znaleźć pliku .sql w assets.